nixos/configuration.nix

```diff
@@ -1,5 +1,10 @@
 { config, pkgs, ... }:
 
+let
+  username = (import ./user.nix).name;
```

Comment:
is it possible to import the file only once, assign it to a variable and then do `user.name` instead of `(import ./user.nix).name`???